javaslang/src/test/java/javaslang/collection/VectorPropertyTest.java

```diff
@@ -0,0 +1,251 @@
+/*     / \____  _    _  ____   ______  / \ ____  __    _______
+ *    /  /    \/ \  / \/    \ /  /\__\/  //    \/  \  //  /\__\   JΛVΛSLΛNG
+ *  _/  /  /\  \  \/  /  /\  \\__\\  \  //  /\  \ /\\/ \ /__\ \   Copyright 2014-2016 Javaslang, http://javaslang.io
+ * /___/\_/  \_/\____/\_/  \_/\__\/__/\__\_/  \_//  \__/\_____/   Licensed under the Apache License, Version 2.0
+ */
+package javaslang.collection;
+
+import javaslang.Function2;
+import javaslang.Tuple;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+import java.util.function.Function;
+
+import static javaslang.Function2.constant;
+import static org.assertj.core.api.Assertions.assertThat;
+
+// TODO should rewrite javaslang-test, once it's accessible from here
```

Comment:
I can't handle it before my vacation. I will file an issue and assign me...

I think it is not possible to have javaslang-test as Maven test-dependency within javaslang (core) because it will lead to a cyclic dependency. Moving javaslang-test into javaslang is also no option. Maybe we can create a separate Maven module that contains only tests - we will see...